File: IDE-BUILD/workspace/hello.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

// Sample C++ file
int main() {
    cout << "Hello, World!" << endl;
    cout << "Welcome to CodeCraft IDE!" << endl;
    
    // Example with vectors
    vector<int> numbers = {1, 2, 3, 4, 5};
    
    cout << "Numbers: ";
    for (int num : numbers) {
        cout << num << " ";
    }
    cout << endl;
    
    // Example with user input
    string name;
    cout << "Enter your name: ";
    getline(cin, name);
    cout << "Hello, " << name << "! Nice to meet you." << endl;
    
    return 0;
}

File: IDE-BUILD/workspace/hello.py
# Sample Python file
print("Hello, World!")
print("Welcome to CodeCraft IDE!")

# Example function
def factorial(n):
    if n <= 1:
        return 1
    return n * factorial(n - 1)

print("Factorial calculations:")
for i in range(1, 6):
    print(f"{i}! = {factorial(i)}")

# Example with user input
name = input("Enter your name: ")
print(f"Hello, {name}! Nice to meet you.")
